### PR TITLE
Removed Ginkgo in require_package_test.go

### PR DESCRIPTION
**Pull request by matthew-hagemann**
This PR:
Aims to align tests with guidelines in #188 
Let me know if I misunderstood any and I can make adjustments.

Closes:
#196

### CODE REVIEW

File: internal/resolution/variable_sources/required_package/required_package_test.go [hunk 1]
```diff
@@ -15,90 +14,80 @@ import (
...
+			property.TypePackage: `{"packageName": "test-package", "version": "2.0.0"}`,
+			property.TypeChannel: `{"channelName":"stable","priority":0}`,
+		})),
+		olmentity.NewBundleEntity(input.NewEntity("bundle-3", map[string]string{
+			property.TypePackage: `{"packageName": "test-package", "version": "3.0.0"}`,
+			property.TypeChannel: `{"channelName":"stable","priority":0}`,
+		})),
+	}
+	rpv = required_package.NewRequiredPackageVariable(packageName, bundleEntities)
+	t.Run("ShouldReturnCorrectPackageName", func(t *testing.T) {
```
Comment (ncdc) [inline internal/resolution/variable_sources/required_package/required_package_test.go:40]: I would suggest that you don't do a literal translation of `It()` to `t.Run()`. It's ok to have multiple assertions in a single block of testing code that aren't each in their own t.Run subtest.

Comment (perdasilva) [reply internal/resolution/variable_sources/required_package/required_package_test.go:40]: +1 and if we could use testify for the assertions, even better ^^
@matthew-hagemann thanks so much for this!

**Comment by ncdc** [comment]
@matthew-hagemann hi, do you think you'll have time to come back to this?

**Comment by tmshort** [comment]
I believe this was resolved in #402.

**Pull request closed without merging**